Projects.js

```diff
@@ -0,0 +1,101 @@
+const express=require('express');
+const db=require('./data/helpers/projectModel');
+const router = express.Router();
+
+
+router.get('/', (req, res) => {
+    db.get()
+    
+     .then(projects => {
+        res.status(200).json(projects)
+          }
+     )
+    .catch(err => {
+        res.status(500).json({
+          message: 'Could not retrieve projects.',
+          err
+        })
+      })
+     })
+
+    router.get('/:id', (req, res) => {
+    const { id } = req.params;
+    db.get(id)
+
+        .then(project => {
+        res.status(200).json(project)
+            }
+        )
+    .catch(err => {
+        res.status(500).json({
+            message: 'Could not retrieve project.',
+            err
+        })
+        })
+        })
+      
+  
+
+router.post('/', (req, res) => {
+    const { object } = req.body;
```

Comment:
I see that you're destructuring the object out of the req.body, but I think what is happening is not what you're expecting to happen.